discogs-labels.csv

```diff
@@ -383,3 +383,5 @@ Label ID, Label Name, Link
 1326, Network Records, https://www.discogs.com/label/1326-Network-Records
 110913, LuckyMe, https://www.discogs.com/label/110913-LuckyMe
 7357, 4 Track Records, https://www.discogs.com/label/7357-4-Track-Records
+6269, Loöq-Records, https://www.discogs.com/label/6269-Loöq-Records
```

Comment:
> @stephanebruckert seems it stuck again :(
Looks like it didn't like the hyphen in the title https://github.com/mirrorfm/data/commit/495fe1e6622b1fbf54323010ceb54fbc243c962e

Loöq-Records => Loöq Records

In the second column just make sure to use the exact title from the label page

Reply:
Thanks